server/chat-commands/core.ts

```diff
@@ -1645,6 +1645,64 @@ export const commands: Chat.ChatCommands = {
 		}
 	},
 
+	requestpartner(target, room, user) {
+		target = this.splitTarget(target);
```

Comment:
id make this and any other commands that might use it use the new `splitUser` stuff

Reply:
I'll do that when it's merged.

Reply:
Why wait until after, especially if this might get merged first? The function already exists in the main code, so there's no reason to not do it here

Reply:
Yeah, all my function does is rewrite the uses of the deprecated version

Reply:
Ah. I thought your PR added it as well.